src/autometrics/decorator.py

```diff
@@ -142,16 +162,23 @@ def async_decorator(func: Callable[P, Awaitable[T]]) -> Callable[P, Awaitable[T]
         @wraps(func)
         async def async_wrapper(*args: P.args, **kwds: P.kwargs) -> T:
             start_time = time.time()
-            caller = caller_var.get()
-            context_token = None
+            caller_module = caller_module_var.get()
+            caller_function = caller_function_var.get()
+            context_token_module = None
```

Comment:
Why is there no type definition here while above there is?
```
context_token_module: Optional[Token] = None
```

Same applies to the context_token_function line below

Reply:
well noted! i forgor 💀